reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md

```diff
@@ -16,18 +16,18 @@ Encrypts content by using the Cryptographic Message Syntax format.
 ## SYNTAX
 
 ### ByContent (Default)
-```
+```PowerShell
```

Comment:
Syntax blocks should not have code language labels. They are not valid PowerShell. See https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/FORMATTING-CODE.md#syntax-block